src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/FlashMessage/flashMessages.html.twig

```diff
@@ -0,0 +1,14 @@
+{% set flashes = app.session.flashbag.all %}
```

Comment:
Thanks for extracting the flash message into a partial of its own! I see you removed the `card messages` class from the container `div` element. Was that intentional? As they seem to have changed the look and feel of the messages. Either use the old look and feel. Or clean up the now unused classes from the SCSS stylesheet.

Reply:
On closer inspection, the card messages aren't that bad after all and line up nicely with the rest of the page. So revert to the old look and feel.